src/processing_pipeline/gemini_2_5_pro_transcription_generator.py

```
@@ -27,32 +34,42 @@ def run(cls, audio_file, gemini_key):
+                            category=HarmCategory.HARM_CATEGORY_CIVIC_INTEGRITY,
+                            threshold=HarmBlockThreshold.BLOCK_NONE,
+                        ),
                     ],
                 ),
             )
 
             if not result.text:
-                raise ValueError("No content in response - likely truncated or blocked")
+                finish_reason = result.candidates[0].finish_reason
+                if finish_reason == FinishReason.MAX_TOKENS:
+                    raise ValueError("The response from Gemini was too long and was cut off.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)